hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -263,9 +262,7 @@ public Void call() throws Exception {
     // Compute the common initial digest for all keys without their UUID
     if (validateWrites) {
       commonInitialMD = DigestUtils.getDigest(DIGEST_ALGORITHM);
-      int uuidLength = UUID.randomUUID().toString().length();
-      keySize = Math.max(uuidLength, keySize);
-      for (long nrRemaining = keySize - uuidLength; nrRemaining > 0;
+      for (long nrRemaining = keySize; nrRemaining > 0;
```

Comment:
should we add some parameter check before processing nrRemaining?
E.g., keySize > 0, keySize > bufferSize, otherwise, if someone specify keySize < 0 or keySize< bufferSize, the nrRemaining and curSize can still be possible be negative.

Reply:
Yes, we don't enter inside for loop if keySize passed is negative.

If we want to add keySize checks, i think it might be applicable for all the parameters like numVolumes, numBuckets. I will open a new jira to handle this.

So, if any parameter is incorrect, show the error message to the user or any other idea how to handle this?

Reply:
Opened https://issues.apache.org/jira/browse/HDDS-1883 for this.

Reply:
LGTM, +1.